### PR TITLE
make seperator between key and value configurable via an env variable

### PR DESCRIPTION
**Pull request by elkowar**
This PR adds the option to configure the seperator between keys and values using the `separator` environment variable

### CODE REVIEW

File: fet.sh [hunk 1]
```diff
@@ -100,7 +100,7 @@ read -r model < /sys/devices/virtual/dmi/id/product_name
 [ $# -gt 0 ] && pkgs=$#
 
 print() {
-	[ "$2" ] && printf '\033[3%sm%6s\033[0m ~ %s\n' "${accent:-4}" "$1" "$2"
+	[ "$2" ] && printf '\033[3%sm%6s\033[0m '"${seperator:-~}"' %s\n' "${accent:-4}" "$1" "$2"
```
Comment (6gk) [inline fet.sh:103]: Don't use variables in the printf sequence, if you want it to accept escape codes use `%b` instead of `%s`.

Comment (6gk) [reply fet.sh:103]: Also it's misspelt, it should be sep**a**rator instead of seperator

File: fet.sh [hunk 2]
```diff
@@ -100,7 +100,7 @@ read -r model < /sys/devices/virtual/dmi/id/product_name
 [ $# -gt 0 ] && pkgs=$#
 
 print() {
-	[ "$2" ] && printf '\033[3%sm%6s\033[0m ~ %s\n' "${accent:-4}" "$1" "$2"
+	[ "$2" ] && printf '\033[9%sm%6s\033[0m%b%s\n' "${accent:-4}" "$1" "${separator:-" ~ "}" "$2"
```
Comment (6gk) [inline fet.sh:103]: > "${separator:-" ~ "}"

You don't need to quote inside the quote.

Comment (elkowar) [reply fet.sh:103]: it looks so weird without it tho :/ but okay i guess

Comment (6gk) [reply fet.sh]: disagree

**Comment by 6gk** [comment]
Thank you!

**Pull request merged**